gui/default/index.html

```diff
@@ -450,23 +450,23 @@ <h4 class="panel-title">
               </div>
               <div class="panel-footer">
                 <button type="button" class="btn btn-sm btn-danger pull-left" ng-click="override(folder.id)" ng-if="folderStatus(folder) == 'outofsync' && folder.type == 'readonly'">
-                  <span class="fa fa-arrow-circle-up"></span>&nbsp;<span translate>Override Changes</span>
+                  <span class="fas fa-arrow-circle-up"></span>&nbsp;<span translate>Override Changes</span>
                 </button>
                 <span class="pull-right">
                   <button ng-if="!folder.paused" type="button" class="btn btn-sm btn-default" ng-click="setFolderPause(folder.id, true)">
-                    <span class="fa fa-pause"></span>&nbsp;<span translate>Pause</span>
+                    <span class="fas fa-pause"></span>&nbsp;<span translate>Pause</span>
                   </button>
                   <button ng-if="folder.paused" type="button" class="btn btn-sm btn-default" ng-click="setFolderPause(folder.id, false)">
-                    <span class="fa fa-play"></span>&nbsp;<span translate>Resume</span>
+                    <span class="fas fa-play"></span>&nbsp;<span translate>Resume</span>
                   </button>
                   <button type="button" class="btn btn-default btn-sm" ng-click="restoreVersions.show(folder.id)" ng-if="folder.versioning.type">
-                    <span class="fa fa-undo"></span>&nbsp;<span translate>Versions</span>
+                    <span class="fas fa-undo"></span>&nbsp;<span translate>Restore</span>
                   </button>
```

Comment:
Why the word change here?

Reply:
In my opinion "Restore" is better suited because it describes what the button does and it is an action like "Pause", "Resume" and "Edit"